### PR TITLE
feat(global): #MED-275 add favorite global resource

### PR DESCRIPTION
**Pull request by Mattox0**
## Describe your changes

Now we can add a favorite global resource and manage them

## Checklist tests

- Add/Remove favorite global resource

## Issue ticket number and link

[MED-275](https://jira.support-ent.fr/browse/MED-275)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: backend/src/main/java/fr/openent/mediacentre/service/GlobalResourceService.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public interface GlobalResourceService {
      * @return Future of all globals ressources
      * @param profile   list all global ressources who match with this profile
      */
-    public Future<List<GlobalResource>> list(Profile profile);
+    public Future<JsonArray> list(Profile profile);
```
Comment (FlorentMr) [inline backend/src/main/java/fr/openent/mediacentre/service/GlobalResourceService.java:26]: Obligé JsonArray? C'est cool le typage Java 😔

File: frontend/src/routes/app/index.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ export const App = () => {
 
   useEffect(() => {
     let newExternalResourcesData: ExternalResource[] = [];
-    if (user?.type.length === 1 && user.type.includes("Relative")) {
+    if (user?.type === "PERSRELELEVE") {
```
Comment (ThomasSevagen) [inline frontend/src/routes/app/index.tsx:73]: wtf is that

Comment (FlorentMr) [inline frontend/src/routes/app/index.tsx:73]: AVC ou signification compliquée?

**Review by FlorentMr — APPROVED** [review]

File: frontend/src/routes/app/index.tsx (hunk 2 above)
Comment (Mattox0) [reply frontend/src/routes/app/index.tsx:73]: Edifice à changer le nommage des types de personnes, ici c'est un parent

Comment (Mattox0) [reply frontend/src/routes/app/index.tsx:73]: Edifice à changer le nommage des types de personnes, ici c'est un parent

**Review by qperie — APPROVED** [review]

**Pull request merged**